src/map/lua/luautils.cpp

```diff
@@ -2765,7 +2765,7 @@ namespace luautils
                     lua_getglobal(LuaHandle, "onMobDeath");
                     if (lua_isnil(LuaHandle, -1))
                     {
-                        ShowError("luautils::onMobDeath: undefined procedure onMobDeath\n");
+                        ShowError("luautils::onMobDeath: undefined procedure onMobDeath in %s/mobs/%s.lua\n", PMob->loc.zone->GetName(), PMob->GetName());
```

Comment:
Was trying to clip `scripts\zones\` off to shorten the message, but have amended it now to use the var.